compiler/control/OMROptions.cpp

```diff
@@ -702,7 +702,7 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
    {"enableGPU=",                          "L{regex}\tlist of additional GPU options: enforce, verbose, details, safeMT, enableMath",
         TR::Options::setBitsFromStringSet, offsetof(OMR::Options, _enableGPU), 0, "F"},
    {"enableGRACostBenefitModel",          "O\tenable GRA cost/benefit model", SET_OPTION_BIT(TR_EnableGRACostBenefitModel), "F" },
-   {"enableGuardedCountingRecompilation", "O\tenable GCR.  If you don't know what that is, I don't have room to explain it here.", RESET_OPTION_BIT(TR_DisableGuardedCountingRecompilations), "F"},
+   {"enableGuardedCountingRecompilation", "O\tinserts recompilation counters at the beginning of methods when their original compilations finish early", RESET_OPTION_BIT(TR_DisableGuardedCountingRecompilations), "F"},
```

Comment:
Hm I know this is a mirror of the `disableGuardedCountingRecompilation` options description, but on a closer read I don't think this is technically accurate.

GCR is adding a recomp counter at the beginning of the method to trigger a recompilation, but the code to decrement the counter is **guarded** to allow the compilation infra to determine whether it wants recompilations to get triggered or not.

Reply:
Hopefully the new explanation is a little more accurate